templates/webserver/webserver-deployment.yaml

```diff
@@ -76,6 +76,19 @@ spec:
           env:
           {{- include "standard_airflow_environment" . | indent 10 }}
       containers:
+        {{- if .Values.authSidecar.enabled }}
+        - name: nginx
+          image: {{ .Values.authSidecar.image }}
+          imagePullPolicy: IfNotPresent
+          ports:
+          - containerPort: 8084
+            name: nginx-ui
```

Comment:
Not sure I have a solid alternative here, but `nginx-ui` doesn't feel right. Maybe `auth-proxy`?

Reply:
this change has been added.